spec/features/welcome/index_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+describe "As a visitor", type: :feature do
+    it "can see a nav bar with link to the welcome page, all items, merchants, shopping carts, login/registration" do 
+
+        visit "/"
+
+        within 'nav' do
+            click_link 'Home Page'
+        end
+
+        expect(current_path).to eq('/')
+        
+        within 'nav' do
+            click_link 'All Items'
+        end
+
+        expect(current_path).to eq('/items')
+
+        within 'nav' do
+            click_link 'All Merchants'
+        end
+
+        expect(current_path).to eq('/merchants')
+
+        within 'nav' do 
+            click_link "Login"
+        end
+
+        expect(current_path).to eq('/login')
+
+        within 'nav' do
+            click_link "Register"
+        end
+
+        expect(current_path).to eq('/register')
```

Comment:
Nice work here! You can just add this (lines 26-36) to the existing navbar test under `navigation_spec.rb`

Reply:
Yeah, I was thinking about that as well, so I will go ahead and do that now